pkg/metrics/metrics.go

```diff
@@ -214,6 +224,15 @@ func UpdateClusterStatusCapacityMaxCount(provider string, region, instanceType,
 	clusterStatusCapacityMaxMetric.With(labels).Set(count)
 }
 
+// UpdateKasFleetManagerClusterProviderResourceQuotaConsumed - sets quota an organisation is currently using
+func UpdateKasFleetManagerClusterProviderResourceQuotaConsumed(quotaId, provider string, count int) {
```

Comment:
Move this closer to the metric definition it's updating. Best to keep them together to find related uses of the metric easier.

Reply:
update the description to replace `organisation` by `user`.

[optional] change `provider` -> `clusterProvider` to make it clear what provider this method is looking for.